tests/samples/test_fake.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from virtool.samples.fake import create_fake_sample, create_fake_samples
+from virtool.fake.wrapper import FakerWrapper
+from virtool.samples.db import LIST_PROJECTION
+
+
+@pytest.fixture
+def app(dbi, pg, run_in_thread, tmpdir):
+    return {
+        "db": dbi,
+        "fake": FakerWrapper(),
+        "pg": pg,
+        "run_in_thread": run_in_thread,
+        "settings": {"data_path": str(tmpdir)},
+    }
+
+
+def clean_sample_document_for_snapshot(fake_sample):
+    del fake_sample["created_at"]
+    for f in fake_sample["files"]:
+        del f["uploaded_at"]
+
+    if "reads" in fake_sample:
+        for f in fake_sample["reads"]:
+            del f["uploaded_at"]
+
+    return fake_sample
+
+
+@pytest.mark.parametrize("paired", [True, False])
+@pytest.mark.parametrize("finalized", [True, False])
+async def test_create_fake_unpaired(paired, finalized, app, snapshot):
```

Comment:
Using `static_time` fixture will make all calls to `virtool.utils.timestamp` return the same value and support snapshotting without have to strip keys from the sample document.

Reply:
Very nice! That saves alot of pain.